chat/src/logic/message.rs

```diff
@@ -65,7 +73,11 @@ impl Message {
     pub fn timestamp(&self) -> &DateTime<Utc> {
         match self {
             Message::AboutMe { timestamp, .. } => timestamp,
+            Message::SchnorrKey { timestamp, .. } => timestamp,
             Message::Chat { timestamp, .. } => timestamp,
+            Message::Unknown => {
+                panic!("unknown message type")
```

Comment:
We prob don't want to panic here? Otherwise older version of the app will panic on load and never update.

Reply:
true